Assets/Scripts/Controllers/TradeController.cs

```diff
@@ -104,7 +104,7 @@ private void TrasfertTradedItems(Trade trade, Vector3 tradingCoordinates)
             if (tradeItem.TradeAmount > 0)
             {
                 Tile tile = WorldController.Instance.World.GetFirstTileWithNoInventoryAround(6, (int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
-                Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount, tradeItem.TradeAmount);
+                Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount);
```

Comment:
What will happen if TradeAmount is > the the max stack size?

This second TradeAmount was there to force the stack size to be big on one tile, if you buy 3000 ice, you have them all in one stack for now.